mmv1/third_party/terraform/go.mod.tmpl

```diff
@@ -1,6 +1,8 @@
 module github.com/hashicorp/terraform-provider-google
 
-go 1.21
+go 1.22.0
```

Comment:
@BBBmau looks like 1 Go 1.22 feature is being used in these dependencies, which means we need to update the Go version for the whole Google provider (+automation used to push to downstreams, I believe). We should pair on this